src/Nowruz/pages/jobs/detail/applicants/useApplicantAction.tsx

```diff
@@ -277,37 +277,37 @@ export const useApplicantAction = (
     [applicantsList],
   );
 
-  const extractCellId = (cell: Cell<Applicant, unknown>) => {
-    let styleClass = '';
+  // const extractCellId = (cell: Cell<Applicant, unknown>) => {
+  //   let styleClass = '';
 
-    switch (cell.id.split('_')[1]) {
-      case 'name':
-        styleClass = 'w-4/12';
-        break;
-      case 'applied':
-        styleClass = 'w-1/12';
-        break;
-      // case 'points':
-      //   styleClass = 'w-1/12';
-      //   break;
-      case 'location':
-        styleClass = 'w-3/12';
-        break;
-      case 'timezone':
-        styleClass = 'w-2/12';
-        break;
-      // case 'experience':
-      //   styleClass = 'w-1/12';
-      //   break;
-      case 'actions':
-        styleClass = 'w-2/12';
-        break;
-      default:
-        break;
-    }
+  //   switch (cell.id.split('_')[1]) {
+  //     case 'name':
+  //       styleClass = 'w-3/12';
+  //       break;
+  //     case 'applied':
+  //       styleClass = 'w-2/12';
+  //       break;
+  //     // case 'points':
+  //     //   styleClass = 'w-1/12';
+  //     //   break;
+  //     case 'location':
+  //       styleClass = 'w-3/12';
+  //       break;
+  //     case 'timezone':
+  //       styleClass = 'w-2/12';
+  //       break;
+  //     // case 'experience':
+  //     //   styleClass = 'w-1/12';
+  //     //   break;
+  //     case 'actions':
+  //       styleClass = 'w-2/12';
+  //       break;
+  //     default:
+  //       break;
+  //   }
 
```

Comment:
Is there any specific reason that you commented these lines instead of deleting them?

Reply:
I put a comment for you (above).  I think you didn't see ;)

"@marjan-ara I think for the applicant/offered table, we don't need to specify a width for each column because it's better to be automatic like other tables (specific width in different sizes of view does not work) if you agree, I removed comment section, if not I can decrease the width of name to w-3/12 and increase size of applied date to w-2/12 to handle Desinger's feedback (one line date)"

Reply:
@satanmourner Yes, I got it. It's OK. I mean why did you keep them as comments?

Reply:
i should first talk to you then if it's ok I will remove it so if it's ok, I will do it

Reply:
Done